website/cue/reference/releases/0.22.0.cue

```diff
@@ -0,0 +1,767 @@
+package metadata
+
+releases: "0.22.0": {
+	date:     "2022-05-24"
+	codename: ""
+
+	whats_next: [
+		{
+			title: "Removal of legacy buffers"
+			description: """
+				With this release, [we've switched the default for disk buffers to the new `v2`
+				implementation](/highlights/2022-04-06-disk-buffer-v2-stable). Tthis means if you set `type
+				= "disk"` you will get the new buffer implementation. In a future release, we will remove the legacy
+				disk buffers.
+				"""
+		},
+	]
+
+	known_issues: []
+
+	description: """
+		The Vector team is pleased to announce version 0.22.0!
+
+		Be sure to check out the [upgrade guide](/highlights/2022-05-03-0-22-0-upgrade-guide) for breaking changes in
+		this release.
+
+		**Important**: as part of this release, we have promoted the new implemenation of disk buffers (`buffer.type
+		= "disk_v2"`) to the default implementation (`buffer.type = "disk"`). Any existing disk buffers (`disk_v1`
+		or `disk`) will be automatically migrated. We have rigorously tested this migration, but recommend making
+		a back up of the disk buffers (in the configured `data_dir`, typically in `/var/lib/vector`) to roll back if
+		necessary. Please see the [release highlight](/highlights/2022-04-06-disk-buffer-v2-stable) for additional
+		updates about this migration.
+
+		In addition to the new features, enhancements, and fixes listed below, this release adds:
+
+		- [Support for iteration has landed in VRL](/highlights/2022-05-18-vrl-iteration-support). Now you can
+		  dynamically map unknown key/value pairs and fields that are arrays. This replaces some common use cases for
```

Comment:
This line reads a bit awkward to me: "map unknown key/value pairs and fields that are arrays". I'm assuming it's meant to say something like "dynamically map unknown key/value pairs in objects, and items in arrays", or something similar?